uninstaller.nsh

```diff
@@ -1,7 +1,6 @@
 !macro customUnInstall
-    ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j uninstall --yes'
+    ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j service uninstall'
     RMDir /r "$APPDATA\remoteit"
-    RMDir /r "C:\Program Files\remoteit-bin"
```

Comment:
@mauricio0000 We do still want to remove this directory... was there a reason you didn't want to remove this?

Reply:
@JamieRuderman if the user in updating the desktop version, the new version should download again the CLI. would you like that I leave these line?

Reply:
Yes please. I think it's important to remove. Thank you!